lib/bolt/pal.rb

```diff
@@ -297,7 +297,7 @@ def list_plans
         errors.each do |error|
           @logger.warn(error.details['original_error'])
         end
-        plans
+        plans.reject { |plan| get_plan_info(plan.first)['private'] }
```

Comment:
A full parse of the plan here on listing the plans will be expensive from both a computation and IO perspective.